_layouts/default.html

```diff
@@ -5,18 +5,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     {% seo %}
-    <link rel="preconnect" href="https://fast.vidalytics.com"  />
+    <link rel="preconnect" href="https://fast.vidalytics.com" />
     <link rel="stylesheet" href="/assets/css/style.css" />
     <link rel="icon" type="image/svg+xml" href="/assets/images/logo.svg" />
   </head>
   <body>
     <header>
       <div class="container">
-        <a href="/">
-          {% svg /assets/images/logo.svg width=45 %}
-          <div class="h2">DevCraft Academy</div>
-        </a>
-        {% include cta_button.html text="Erstgespräch vereinbaren" %}
+        <div class="navigation">
+          {% include logo_button.html text="In Rekordzeit zum Senior" %} {%
+          include navigation.html %}
+        </div>
```

Comment:
Sehr schön! Das `{%` könnte auch noch in die nächste Zeile wandern.

Reply:
Mobil gibt's hier noch Probleme, das passt bei kleineren Bildschirmen nicht in eine Zeile. Fällt dir da eine Lösung ein?

<img width="443" alt="Bildschirmfoto 2023-10-26 um 16 43 30" src="https://github.com/DevCraft-Academy/landingpage/assets/141632/60d7be79-4ffb-4c1c-a2fe-4ca66f35e38f">

Reply:
ich werde ab einem bestimmten Breakpoint die Elemente untereinander anordnen

Reply:
@clemenshelm Das {% wird ständig wieder zurück in die obere Zeile gesetzt wenn ich speichere...ich kann gerade nicht nachvollziehen woran das liegt

Reply:
@clemenshelm jetzt hat die Formatierung geklappt, hab prettier deaktiviert